### PR TITLE
LPD-13652 Product Specification fragment

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/LPD-13652

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:sf

File: modules/test/playwright/pages/commerce/commerceLayoutsPage.ts [hunk 1]
```diff
@@ -98,6 +121,27 @@ export class CommerceLayoutsPage {
...
+		await source.press('Enter');
+		await source.press('Enter');
+	}
+
+	async addWidgetToPage(widgetName: string) {
+		await this.addWidgetButton.click();
+		await this.searchFormInput.click();
+		await this.searchFormInput.fill(widgetName);
+		await this.addWidgetLabel(widgetName).click();
+	}
```
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/test/playwright/pages/commerce/commerceLayoutsPage.ts:143]: This is your code, I just sorted it

**Comment by andrea-ale-sbarra** [comment]
@alessiorendina overall looks good.
Can you please check my comment, please make sure to not introduce duplicates.
Thank you

File: modules/test/playwright/pages/commerce/commerceLayoutsPage.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (alessiorendina) [reply modules/test/playwright/pages/commerce/commerceLayoutsPage.ts:143]: Also, now that I see it, I can remove L140. The fill() method is enough, there's no need to click on the input.

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce-functional

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:report:2547282522

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:report:2547607125

**Comment by alessiorendina** [comment]
ci:forward:force

**Pull request closed without merging**